R/plotconfiguration-axis.R

```diff
@@ -2,9 +2,18 @@
 #'
 #' @include enum.R
 #' @export
+#' @description
+#'  Built-in transformations include
+#'  "asn", "atanh", "boxcox", "date", "exp", "hms", "identity",
+#'  "log", "log10", "log1p", "log2", "logit", "modulus", "probability",
+#'  "probit", "pseudo_log", "reciprocal", "reverse", "sqrt" and "time"
 Scaling <- enum(c(
+  "lin",
+  "log10",
   "log",
-  "lin"
+  "asn", "atanh", "boxcox", "date", "exp", "hms", "identity",
+  "log1p", "log2", "logit", "modulus", "probability",
+  "probit", "pseudo_log", "reciprocal", "reverse", "sqrt", "time"

```

Comment:
!! WOW.

Reply:
These are all the options available in ggplot, we can select only a few of them if we want